static/src/javascripts-legacy/projects/commercial/modules/dfp/add-slot.js

```diff
@@ -9,27 +9,27 @@ define([
 ], function (mediator, dfpEnv, Advert, queueAdvert, loadAdvert, enableLazyLoad, performanceLogging) {
     return addSlot;
 
-    function addSlot(adSlot) {
+    function addSlot(adSlot, forceDisplay) {
         adSlot = adSlot instanceof HTMLElement ? adSlot : adSlot[0];
 
         if (dfpEnv.firstAdDisplayed && !(adSlot.id in dfpEnv.advertIds)) { // dynamically add ad slot
             // this is horrible, but if we do this before the initial ads have loaded things go awry
             if (dfpEnv.firstAdRendered) {
```

Comment:
I wanted to test whether that assumption still held actually, I don't think it does. It was there before I started working here and I never really questioned this code. Yet it feels like a hack. According to [the GPT reference](https://developers.google.com/doubleclick-gpt/reference#googletag.display), we're doing everything right.